scenes/game/local_game.gd

```diff
@@ -6416,12 +6461,31 @@ func handle_strike_effect(card_id : int, effect, performing_player : Player):
 			if boosts_in_play > 0:
 				performing_player.strike_stat_boosts.speed += effect['amount'] * boosts_in_play
 				events += [create_event(Enums.EventType.EventType_Strike_SpeedUp, performing_player.my_id, effect['amount'] * boosts_in_play)]
+		"spend_all_force_and_save_amount":
+			var force_amount = performing_player.get_available_force()
+			var card_names = ""
+			if performing_player.hand.size() > 0:
+				card_names = ": " + performing_player.hand[0].definition['display_name']
+				for i in range(1, len(performing_player.hand)):
+					card_names += ", " + performing_player.hand[i].definition['display_name']
+			if performing_player.gauge.size() > 0:
+				if card_names == "":
+					card_names += ": "
+				else:
+					card_names += ", "
+				card_names += performing_player.gauge[0].definition['display_name']
+				for i in range(1, len(performing_player.gauge)):
+					card_names += ", " + performing_player.gauge[i].definition['display_name']
+			_append_log_full(Enums.LogType.LogType_CardInfo, performing_player, "spends all cards in hand an gauge to generate %s force%s." % [force_amount, card_names])
+			events += performing_player.discard_hand()
+			events += performing_player.discard_gauge()
+			performing_player.force_spent_before_strike += force_amount
```

Comment:
kind of weird to be +=
I know this doesn't matter, and Vatista can't ever spend force and strike, so even if this variable counted all force spent in a turn it wouldn't matter.